File: src/domain/auth/exception/invalid-verify-code.exception.ts
import { HttpStatus } from '@nestjs/common';

import { ApiException } from '../../../common/utils/swagger';

export class InvalidVerifyCodeException extends ApiException {
  constructor() {
    super({
      swagger: '잘못된 인증 코드일 때',
      message: '올바르지 않은 인증 코드입니다.',
      code: HttpStatus.BAD_REQUEST,
    });
  }
}

File: src/main/java/com/github/kmu_wink/wink_official_backend/domain/auth/exception/InvalidVerifyCodeException.java
package com.github.kmu_wink.wink_official_backend.domain.auth.exception;

import com.github.kmu_wink.wink_official_backend.common.api.exception.ApiException;
import org.springframework.http.HttpStatus;

public class InvalidVerifyCodeException extends ApiException {

    public InvalidVerifyCodeException() {

        super(HttpStatus.BAD_REQUEST, "유효하지 않은 인증 코드입니다.");
    }
}
